src/interpreter_funcs/join_meshes.rs

```diff
@@ -39,10 +39,9 @@ impl Func for FuncJoinMeshes {
     }
 
     fn call(&mut self, args: &[Value]) -> Result<Value, FuncError> {
-        let first_mesh = args[0].unwrap_mesh();
-        let second_mesh = args[1].unwrap_mesh();
+        let meshes = args.iter().map(|a| a.unwrap_mesh());
```

Comment:
Just a note that we don't actually support varargs and this seems a bit courageous to do (will crash as soon as we add a nonmesh parameter here), but will work with the current version, so we can leave it for now.

Reply:
### I'm aware of it and that was also what I was thinking - once we add another arg (if we ever have a reason to do so), we'll have to rework this anyway.